modules/apps/27-interchain-accounts/types/keys.go

```diff
@@ -6,22 +6,34 @@ const (
 	// ModuleName defines the Interchain Account module name
 	ModuleName = "interchainaccounts"
 
-	// Version defines the current version the IBC tranfer
+	// Version defines the current version the IBC interchainaccounts
 	// module supports
 	Version = "ics27-1"
 
+	// PortID is the default port id that the interchainaccounts module binds to
 	PortID = "ibcaccount"
 
-	StoreKey  = ModuleName
+	// StoreKey is the store key string for IBC interchainaccounts
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for IBC interchainaccounts
 	RouterKey = ModuleName
 
+	// QuerierRoute is the querier route for IBC interchainaccounts
+	QuerierRoute = ModuleName
+)
+
+var (
+	// ModuleAccountKey defines the key used to construct to the IBC interchainaccounts module account address
+	ModuleAccountKey = []byte(ModuleName)
+
 	// Key to store portID in our store
 	PortKey = "portID"
 
-	QuerierRoute = ModuleName
-
 	// MemStoreKey defines the in-memory store key
 	MemStoreKey = "mem_capability"
```

Comment:
`MemStoreKey` and `KeyPrefixRegisteredAccount` below both appear to be used. Any reason they can't be removed?

Reply:
Lets remove them. They look outdated